src/stories/SpeedInput.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { ComponentPropsWithoutRef } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import SpeedInputComponent, { SpeedInputProps, SPEED_UNITS, VehicleSpeed } from 'src/SpeedInput/SpeedInput';
+
+export default {
+    title: 'Lakefront/SpeedInput',
+    component: SpeedInputComponent,
+    argTypes: {
+        value: {
+            control: 'VehicleSpeed',
+            table: {
+                defaultValue: { summary: 'VehicleSpeed' },
+                type: { summary: null }
+            }
+        },
+        unitConversionRequired: {
+            control: 'boolean',
+            table: {
+                defaultValue: { summary: true },
+                type: { summary: 'boolean' }
+            }
+        },
+        allowNegativeInput: {
+            control: 'boolean',
+            table: {
+                defaultValue: { summary: true },
+                type: { summary: 'boolean' }
+            }
+        },
+        defaultUnits: {
+            control: SPEED_UNITS,
+            table: {
+                defaultValue: { summary: 'Kph' },
+                type: { summary: 'string' }
+            }
+        },
+        disabled: {
+            control: 'boolean',
+            table: {
+                defaultValue: { summary: false },
+                type: { summary: 'boolean' }
+            }
+        }
+    },
+} as Meta;
+
+const Template: Story<SpeedInputProps & ComponentPropsWithoutRef<'div'>> = (args) => {
+
+    const updateOnChange = (_speedRange: VehicleSpeed | null): void => {
+    };
```

Comment:
Please add the function. May be just create a state and update that state to show a message whenever the change event is triggered. Check Refreshtoolbar implementation.